articles/flutter-vs-kotlin/index.md

```diff
@@ -0,0 +1,91 @@
+Mobile application development has become an essential part of the software business in recent years. Previously, people used to prefer ReactNative or Flutter to develop cross-platform applications due to their low-cost features. But today, Kotlin has managed to enter into the world of cross-platform development.
+
+Since Flutter has been improved a lot in the past years, it now provides better features than ReactNative.  Kotlin has also entered the space of cross-platform development as well.
+
+In this article, we will be comparing the new languages for cross-platform development. The primary goal of this article is to help developers distinguish the best cross-development platform for them, depending on their needs and skills.
+
+### PERFORMANCE
+When it come to mobile app development, both Kotlin and Flutter are each superb with some distinctions. Kotlin can compile the code and customize it to the best needs of its user as well as the native apps.
+
+Flutter allows reloading functionality, giving the developer a choice of selection if all the different elements are performing well.
+
+Developers describe Flutter as one of the most popular cross-platform mobile framework. Kotlin, however, is described as a statistical-type programming language.
+
+### LANGUAGE
+The main advantage of Kotlin is that its programming language is quite similar to Java. Developers who have worked with Java will find it much easier to work with Kotlin. On the other hand, Flutter uses Dart from Google. Although Dart is remarkably used, it is easier to understand and operate.
+
+### PRICE AND TESTING
+Both Kotlin and Flutter are free, but the real difference occurs in testing support. While Kotlin tends to support backend development, Flutter works on extensive testing features.
+
+### UI SUPPORT
+Kotlin as a multi-platform development tool has a unique distinction of lending a freehand to the UI developer to make an app without restrictions and make a truly customized app. On the other hand, developers can take the help of widgets if they are working with Flutter, which also helps in customized solutions.
+
+### PROS OF KOTLIN
+Kotlin is an SDK, so it is not necessary to shift the complete app to the platform. Due to its similarities with Java, developers find it easier to get a hand on it and use a multi-frame to design a cross-platform app. Since UI creation is not restricted, Developers have the freedom to work on any aspect of the app.
+
+### CONS OF KOTLIN
+Very few built-in widgets and libraries are available for developers as Kotlin is still new on the market and is still growing. Once the date-time library arrives, the situation will improve. The development cost is quite high as well, and it will eventually be dearer in the long run. According to [HiddenBrains](https://www.hiddenbrains.com/blog/kotlin-vs-flutter-rule-android.html), the app developer must have highly developed skills to understand each platform and use APIs to handle Kotlin well.
```

Comment:
Please clear this sentence up:

" and it will eventually be dearer in the long run" 

Not sure what dearer was meant to be.